### PR TITLE
update onboarding with links to uilib wiki

### PR DESCRIPTION
**Pull request by ethanshar**

### CODE REVIEW

File: docs/App.UiLib.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ To that end, we are going to use [react-native-ui-lib](https://github.com/wix/re
 
 It was designed to enable developers to build and customize their own visual language so it is not specific in any way to Wix. The Wix mobile app, for example, is using a private UI library, wix-react-native-ui-lib, as a wrapper around react-native-ui-lib. Most of the components come directly from react-native-ui-lib and are customized to fit the purpose and visual language of the Wix mobile app.
 
+Please read more in our uilib's [Wiki Page](https://github.com/wix-private/wix-react-native-ui-lib/wiki) (see table of content)
```
Comment (drorbiran) [inline docs/App.UiLib.md]: This is the open source library so we can't add it in here. The plan is to have 2 separate libraries public and private. This article is only related to the open source.

File: docs/App.UiLib.md [hunk 2]
```diff
@@ -26,18 +28,17 @@ It was designed to enable developers to build and customize their own visual lan
 # Getting Started
 
 ## 1. Install react-native-ui-lib
+[Read here.](https://github.com/wix-private/wix-react-native-ui-lib/wiki/INSTALLATION-&-USAGE)
```
Comment (drorbiran) [inline docs/App.UiLib.md]: Can't add this since it's wix-private

File: docs/App.UiLib.md [hunk 3]
```diff
@@ -54,7 +55,9 @@ That's a rather bad way to do it.
...
 
-Luckily, the ui-lib already comes with a set of predefined constants and presets, which define [colors](https://github.com/wix/react-native-ui-lib/blob/dec607df9c94fd37e3dc6f01b1db6847d24f1155/src/style/colorsPalette.js), [typography](https://github.com/wix/react-native-ui-lib/blob/dec607df9c/src/style/typography.js), [shadows](https://github.com/wix/react-native-ui-lib/blob/dec607df9c/src/style/shadows.js) and more. You can easily use these presets anywhere in your code. It's also very easy to [define your own presets](https://github.com/wix/react-native-ui-lib/wiki/STYLE).
+Luckily, the ui-lib already comes with a set of predefined constants and presets, which define colors, typography, shadows and more. You can easily use these presets anywhere in your code. It's also very easy to [define your own presets](https://github.com/wix/react-native-ui-lib/wiki/STYLE).
+
+Read about the preset and how to use there [here.](https://github.com/wix-private/wix-react-native-ui-lib/wiki/The-Basics)
```
Comment (drorbiran) [inline docs/App.UiLib.md]: private :(

File: docs/App.UiLib.md (hunk 1 above)
Comment (ethanshar) [reply docs/App.UiLib.md]: oy, my bad. 
can we revert?

**Pull request merged**